getting-started/templates/systemlink-values.yaml

```diff
@@ -464,6 +464,7 @@ dataframeservice:
     ##
     service: *minioServiceName
     ## S3 port number.
+    # <ATTENTION> This must be overridden if not using the SLE MinIO instance.
```

Comment:
We already have this comment block above, which sort of duplicates this information. There was still confusion when deploying to GM despite this comment block, so I feel like it's worth adding an extra call-out to the `port` section as well.

```
    ## Set this value to connect to an external S3 instance.
    # <ATTENTION> To connect to an external S3 bucket, set the host here as well as the schemeName and port.
    ##
    host: ""
 ```